src/containers/App.jsx

```diff
@@ -56,59 +70,61 @@ const App = ({ accounts, konnectors, triggers }) => {
   const webviewIntent = useWebviewIntent()
   const theme = useCozyTheme()
 
-  const { shortcutsDirectories } = useCustomShortcuts()
+  const { t } = useI18n()
+
+  const homeMagicFolderConn = mkHomeMagicFolderConn(t)
```

Comment:
I would have left this logic in a custom hook to separate it from the operation of the component all using useQuery in it

Reply:
We discussed this point outside GitHub. To conclude, in this case, as the code is not intended to be reused, it doesn't necessarily make sense to extract it in a hook